core-war/src/main/java/org/silverpeas/web/socialnetwork/myprofil/control/MyProfilSessionController.java

```diff
@@ -141,12 +147,25 @@ public void modifyUser(String idUser, String userLastName, String userFirstName,
       }
 
       // process extra properties
-      for (Map.Entry<String, String> property : properties.entrySet()) {
-        if (theModifiedUser.isPropertyUpdatableByUser(property.getKey()) ||
-            (isAdmin() && theModifiedUser.isPropertyUpdatableByAdmin(property.getKey()))) {
-          theModifiedUser.setValue(property.getKey(), property.getValue());
+      Enumeration<String> parameters = request.getParameterNames();
+      String parameterName;
+      String property;
+      while (parameters.hasMoreElements()) {
+        parameterName = parameters.nextElement();
+        if (parameterName.startsWith("prop_")) {
+          // remove s"prop_"
+          property = parameterName.substring(5, parameterName.length());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS109)